src/main/webapp/profileScript.js

```diff
@@ -632,14 +621,16 @@ function createEventsPostsTab() {
 function displayPosts(user, viewer, postsContainer) {
   let count = 0; 
   const currentId = localStorage.getItem(LOCAL_STORAGE_ID);
+  const postsGrid = document.createElement('div');
+  postsGrid.id = 'posts-grid';
 
   fetch('/post')
     .then(response => response.json())
     .then(posts => {
       if (viewer === PROFILE_VIEWER_PERSONAL) {
         for (let i = 0; i < posts.length; i ++) {
           if (posts[i].creator === user.id) {
-            postsContainer.appendChild(createPostWithResponse(posts[i], currentId));
+            postsGrid.appendChild(createPost(posts[i], currentId));
```

Comment:
I believe I ended up changing the function for creating posts to createPostWithResponse, similar to events